assets/src/components/propertiesPanel/ghostPropertiesPanel.tsx

```diff
@@ -20,6 +38,12 @@ const GhostPropertiesPanel = ({ selectedGhost, route }: Props) => (
       <BlockWaiverList blockWaivers={selectedGhost.blockWaivers} />
     )}
 
+    {featureIsEnabled("block_waivers") &&
+      !hasBlockWaiver(selectedGhost) &&
+      !isLateVehicleIndicator(selectedGhost) && (
```

Comment:
We don't show the Ghost Properties Panel for virtual ghosts. (It shows the VPP for the corresponding real vehicle.) So I don't think you need to check this.

Reply:
Ah, good point. Removed the check.